app/main.py

```diff
@@ -1 +1,16 @@
-# write your code here
+from app.errors import (VaccineError, NotWearingMaskError)
+from app.cafe import Cafe
+
+
+def go_to_cafe(friends: list[dict], cafe: Cafe) -> str:
```

Comment:
According to the checklist, it is a good practice to use type annotations. Please add type annotation for the return value of the function `go_to_cafe`.